<?xml version="1.0" encoding="utf-8"?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title>MariaDB Knowledge Base Comments for: mariabackup failed running as root.</title><link>https://mariadb.com/kb/en/mariabackup-failed-running-as-root/+comments/feed/</link><description></description><atom:link href="https://mariadb.com/kb/en/mariabackup-failed-running-as-root/+comments/feed/" rel="self"></atom:link><language>en-us</language><lastBuildDate>Mon, 13 Jan 2025 10:58:09 +0000</lastBuildDate><item><title>Re: mariabackup failed running as root.</title><link>https://mariadb.com/kb/en/mariabackup-failed-running-as-root/+comments/5624</link><description>&lt;p&gt;This looks like a exception (network error of some sort maybe) was generated in an AWS (proprietary?) plugin related to EC2 storage that wasn't caught. The mariabackup didn't know how to handle this (or its a predefined failure in the C++ standards).&lt;/p&gt;
&lt;p&gt;If you still think its a mariabackup bug please create a JIRA bug report with details.&lt;/p&gt;
</description><dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">Daniel Black</dc:creator><guid>https://mariadb.com/kb/en/mariabackup-failed-running-as-root/+comments/5624</guid></item></channel></rss>